### DISCUSSION TITLE
Support for NEXT ZOOM

### DISCUSSION THREAD

**Discussion by mingsu** [rxhanson/Rectangle#325] [Ideas]
In many cases, one short key is very easy to remember, I suggests a key for NEXT ZOOM.

NEXT ZOOM is depend on the current mode and current location, if the current is:
- first fourth, go to next fourth
- last fourth, go to first fourth
- top left, go to top right
- left half, go to right half
- bottom half, go to up half
- ...

**Comment by rxhanson** [comment]
I like this idea, but would it be better to just have the repeated actions set up for each of these window actions, like it is with fourths? Then you're only executing one shortcut multiple times instead of one shortcut and then the NEXT ZOOM shortcut.
